Manual.md

```diff
@@ -1444,9 +1444,18 @@ Python packages should be built with the `python{,2,3}-module` build style, if p
 This sets some environment variables required to allow cross compilation. Support to allow
 building a python module for multiple versions from a single template is also possible.
 
-To allow cross compilation, the `python-devel` package (for python 2.7) must be added
-to `hostmakedepends` and `makedepends`. If any other python version is also supported,
-for example python3.4, those must also be added as host and target build dependencies.
+Python packages that rely on `python-setuptools` or `python3-setuptools` should generally map
+`install_requires` dependencies in `setup.py` to `depends` in the template, and `setup_requires`
```

Comment:
While you are here, could you add: ".. `depends` in the template, or `python3` if there are none."?